helm/data/values/prod/values.yaml

```diff
@@ -1,12 +1,6 @@
-imagePullPolicy: IfNotPresent
```

Comment:
Why is this removed?

Reply:
This is [the default value in the chart](https://github.com/dhis2-sre/im-inspector/blob/master/helm/chart/values.yaml#L18) already, so I'm just making it consistent with the `im-manager` helm setup.